src/arch/x86.rs

```diff
@@ -0,0 +1,76 @@
+#[cfg(target_arch = "x86")]
+use std::arch::x86::*;
+#[cfg(target_arch = "x86_64")]
+use std::arch::x86_64::*;
+
+
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[target_feature(enable = "sse")]
+#[target_feature(enable = "fma")]
+pub unsafe fn _merge_impl(original: crate::pixel::Rgba, other: crate::pixel::Rgba) -> crate::pixel::Rgba {
```

Comment:
- Instead of absolute path `crate::pixel::Rgba` import `crate::Rgba` and use `Rgba` directly
- `original` should be renamed to `base`

Reply:
yeah see was lazy but ok 3332